libs/EXTERNAL/libtatum/libtatum/tatum/graph_visitors/CommonAnalysisVisitor.hpp

```diff
@@ -328,6 +328,15 @@ bool CommonAnalysisVisitor<AnalysisOps>::do_arrival_traverse_edge(const TimingGr
 
                 for(const TimingTag& src_capture_clk_tag : src_capture_clk_tags) {
                     //Standard propagation through the clock network
+
+                    //Skip propagation of timings derived from incompatible constraints
+                    if ( tg.node_type(sink_node_id) == NodeType::CPIN ) {
+                        if ((tg.trigg_edge(sink_node_id) == 1 && !tc.clock_domain_inverted(src_capture_clk_tag.launch_clock_domain())) ||
```

Comment:
Added enums and comment
`non-inverted clock`  refers to netlist clocks constrained in SDC file with create_clock command.
`inverted clocks` are based on netlist clocks. They are created as virtual clocks and are 180 degree shifted in relation to original netlist clock. I believe that current solution is capable of representing rise-rise, rise-fall, fall-rise and fall-fall transfers because the tags are propagated from input clock sources to different nodes describing the FFs clocked at falling or rising edge.
For example you can look at this TimingGraph:
![timing_graph](https://user-images.githubusercontent.com/33930985/209343878-04eb2e8b-bc7e-4251-84ec-0e292b550831.png)
It was generated from `multiclock.blif` cricuit with `FFC` latch configured as FF clocked at falling edge (represented on TimingGraph with nodes: 6, 10, 24). FFC uses the same clock (input clock source, node 3 ) as FFA (nodes: 5, 20, 9) .
Tags from the `inverted clock` were propagated to FFC nodes, while tags from `non-inverted clock` were propagated to FFA.